src/Utils/PrettyPrint.hs

```diff
@@ -70,15 +73,15 @@ prettyFrom f (Fi e (l1, ()) (l2, ())) =
   [ "fi " ++ prettyExpr e
   , "\tfrom " ++ f l1
   , "\telse " ++ f l2]
-prettyFrom _ (Entry ()) = ["entry"]
+prettyFrom _ (Entry p ()) = ["entry( " ++ prettyPat p ++ ")"]
```

Comment:
Nu er vi ikke helt færdig med grammatikken, men ideelt for mig vil det være:
`"entry " ++ prettyPat p`
Samme med exit